consensus/crdt/config.go

```diff
@@ -87,6 +116,10 @@ func (cfg *Config) Validate() error {
 	if cfg.RebroadcastInterval <= 0 {
 		return errors.New("crdt.rebroadcast_interval is invalid")
 	}
+
+	if cfg.Batching.MaxQueueSize <= 0 {
+		return errors.New("crdt.batching.max_queue_size is invalid")
+	}
```

Comment:
Also validate `MaxBatchSize` is less than `MaxQueueSize`?

Reply:
They are not related. There is a goroutine building the batches from the information that is put on a channel and commiting when the time comes. MaxQueueSize is the buffer size of that channel. Regardless of the MaxBatchSize, the queue will only get filled if the goroutine is unable to batch-add+commit things as fast as the requests arrive.

I can think of MaxBatchSize = 5000 while MaxQueueSize is left at the default of 1000 and that should work, I guess.

Reply:
This made me think and I bumpbed the default MaxQueueSize to 50000. I don't think this limit should be hit during normal operation regardless of maxBatchSize.

Reply:
Makes sense!